vmm/src/memory_manager.rs

```diff
@@ -1621,12 +1617,20 @@ impl MemoryManager {
         match self.hotplug_method {
             HotplugMethod::VirtioMem => {
                 if desired_ram >= self.boot_ram {
```

Comment:
In fact you could pull this up to the top of the function too (like my suggested change for CpuManager.)

Reply:
Can I check for `self.boot_ram` on both VirtioMem and Acpi hotplug methiods?

Reply:
Yes, I think so!